core/templates/components/ck-editor-helpers/ck-editor-copy-content-service.ts

```diff
@@ -70,14 +71,16 @@ export class CkEditorCopyContentService {
         break;
       }
 
-      if (currentElement.parentElement.tagName === this.OUTPUT_VIEW_TAG_NAME) {
+      if (currentElement.parentElement.tagName === this.OUTPUT_VIEW_TAG_NAME ||
+        currentElement.tagName === this.OUTPUT_VIEW_TAG_NAME) {
```

Comment:
Can you extract this logic to a separate function to make the code more readable?

Reply:
I can do this, but it increases code duplication heavily. Are you sure that's what we want? The only reason this line was added was to prevent the "Cannot read property tagName of null" error.

Reply:
How it increases code duplication? What would we duplicate?

Reply:
Code duplication occurs because the body of the if statement will remain same even within the separate function - I don't think that enhances readability. I can add more inline comments to explain the condition better, that could definitely be worked out.

Reply:
That is not code duplication. Having the logical statement in a separate function enables you to name the function in a way that when you put it into the `if` it si more readable. 
```
if (isElemOrParentElemOutputView(currentElement)) {
   break;
}

Reply:
I'll separate the logic into an alternate function. Now that I figure it out, it should help with the tests too.

Reply:
I've refactored it, can you have a look?